ui/shared/utils/constants.js

```diff
@@ -590,6 +591,66 @@ const ORDERED_VEP_CONSEQUENCES = [
     value: 'INV',
     group: VEP_GROUP_SV,
   },
+  {
+    description: 'A loss of function effect',
+    text: 'Loss of function',
+    value: 'LOF',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'A loss of function effect via intragenic exonic duplication',
+    text: 'Loss Of Functionn Duplication',
```

Comment:
typo in you have an extra `n` in `Functionn`, and also should be `Loss Of Function - Duplication`